accelerator/cuda_accelerator.py

```diff
@@ -147,6 +147,9 @@ def is_fp16_supported(self):
         else:
             return False
 
+    def supported_dtypes(self):
+        return [torch.float, torch.half]
```

Comment:
Cuda support `torch.bfloat16` as well, right?

Reply:
Is there any reason we can not use the data types defined here
https://pytorch.org/docs/stable/tensors.html#data-types

Reply:
Hi tjruwase, I am sorry for the late reply.  Cuda support torch.bfloat16 as well. I will update my PR later, Thanks~